### PR TITLE
Add format func

### PR DESCRIPTION
**Pull request by viet-wego**
Add a `func` to format the amount.

Example:

```
str := currency.Format(2504.2023, "USD", "en")
fmt.Println(str)
// expected output: $2,504.20
```

### CODE REVIEW

File: currency/currency_test.go [hunk 1]
```diff
@@ -159,3 +159,79 @@ func Test_FromMinorUnit_ZeroAmount(t *testing.T) {
...
+			amount:       2048.179,
+			currencyCode: "USD",
+			locale:       "",
+		},
+		"Invalid locale returns in en locale": {
+			amount:       2048.179,
+			currencyCode: "USD",
+			locale:       "qwertyuiop",
+			want:         "$2,048.18",
+		},
```
Comment (yanyi-wego) [inline currency/currency_test.go:188]: Wonder if empty locale should fallback to `en` locale like the invalid locale case. Thoughts?

**Review by yanyi-wego — DISMISSED** [review]

**Review by aaron-wego — DISMISSED** [review]

File: currency/currency_test.go (hunk 1 above)
Comment (aaron-wego) [reply currency/currency_test.go:188]: what is the default? I think in the library they return some default

Comment (yanyi-wego) [reply currency/currency_test.go]: They return [empty struct based on their code](https://github.com/bojanz/currency/blob/4d4f7633e8bc01f21d170dd31e6a405055400e42/locale.go#L24).

https://go.dev/play/p/7fbAj85FnbK

**Review by yanyi-wego — APPROVED** [review]
Re-approve after implementing fallback to `en` locale if empty string.

**Review by aaron-wego — APPROVED** [review]

**Pull request merged**